### ISSUE TITLE
DevRecT1 - Fixture setup for testing (and potentially dev setup)

### ISSUE THREAD

**Issue by TheMadBug** [SACGF/variantgrid#1167]
Best to wait on #1126
Also maybe on #1166 - so that each main module would have its own fixture set

Create full set of fixures that can be used for testing - or even development.

Need to work out if we migrate existing test plans / leave existing test plans as is / start fresh etc
Also write a few more test plans to make sure fixture is comprehensive

**Comment by davmlaw** [comment]
See also - #736

I usually try and compress all the data migrations every so often. Might be easier to dump SQL into CSV then write them out

**Comment by davmlaw** [comment]
🤖 Written by Claude

Largely covered: the test fixture builders are indexed in `claude/guides/testing.md` and CI runs the suite (`.github/workflows/django-tests.yml`). The only part not done is a dev seed dataset. If that's wanted, it's better as its own issue.
